django/contrib/postgres/lookups.py

```diff
@@ -60,3 +60,24 @@ def process_lhs(self, qn, connection):
             self.lhs = SearchVector(self.lhs)
         lhs, lhs_params = super(SearchLookup, self).process_lhs(qn, connection)
         return lhs, lhs_params
+
+
+class TrigramSimilar(PostgresSimpleLookup):
+    lookup_name = 'trigram_similar'
+    operator = '%%'
+
+
+class TrigramDistance(PostgresSimpleLookup):
```

Comment:
The function is definitely useful, but I'm not sure that it should exist as a lookup. There are no other lookups in Django which take multiple arguments like this one does. Expressions are currently the right API for multi parameter functions.

Also I'm pretty sure `foo__trigram_distance((F('bar'), F('closeness'))` won't work, whereas you'd get this for free with the expression version.